Dockerfile

```diff
@@ -15,11 +15,12 @@ RUN mv /powershell/PSDepend/0.1.64/PSDependScripts/Noop.ps1 /powershell/PSDepend
 ## base image
```

Comment:
Should bump the image version in `version.txt`

Reply:
oops, will do